### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'get_available_nnet_and_data_packets'

### ISSUE THREAD

**Issue by itsderek23** [luxonis/depthai-python-extras#1]
If I run:

```
python test_cnn.py
```

and `CTRL-C` a couple of times to exit the script, eventually I get the following error when attempting to run the script again:

```
pi@raspberrypi:~/projects/DepthAI/python-api $ python3 test_cnn.py
depthai.__version__ == 0.0.4a
CNN configurations read: ./resources/mobilenet_ssd.json
Changing streaminfo sizes to: 300x300x3
depthai: before xlink init;
Device Found name 1.2- 
about to boot device with "cmd_file": ./depthai_ai.cmd
not xlink success
depthai: Error initalizing xlink;
Traceback (most recent call last):
  File "test_cnn.py", line 43, in <module>
    nnet_packets, data_packets = p.get_available_nnet_and_data_packets()
AttributeError: 'NoneType' object has no attribute 'get_available_nnet_and_data_packets'
```

I'm assuming the Python code breaks because of:

```
not xlink success
depthai: Error initalizing xlink;
```

...which might be triggered by not gracefully exiting the script.

System info:

```
$ ./log_system_information.sh 
Linux raspberrypi 4.19.66-v7+ #1253 SMP Thu Aug 15 11:49:46 BST 2019 armv7l GNU/Linux
depthai.__version__ == 0.0.4a
numpy.__version__ == 1.16.2
Python 3.7.3
```

**Comment by itsderek23** [comment]
This doesn't happen on every `CTRL-C` (more frequently, things cleanup fine). 

Clicking the `module rst` button is the fastest way to get things working again w/the DepthAI: RPi Compute Module Edition.

**Comment by itsderek23** [comment]
Fixed in #3.

**Comment by itsderek23** [comment]
Likely fixed via #30.
